tests/infra/e2e_args.py

```diff
@@ -255,7 +255,12 @@ def cli_args(add=lambda x: None, parser=None, accept_unknown=False):
         default=None,
     )
     parser.add_argument(
-        "--long-tests", help="Enable extended tests", action="store_true", default=False
+        "--long-tests", help="Enable extended tests", action="store_true"
```

Comment:
Does this default to `False`? It might be worth leaving it the default value to be more explicit here?

Reply:
That's always the case for store_true